sig-community-experience/charter.md

```diff
@@ -0,0 +1,94 @@
+# SIG YOURSIG Charter
+* prefilled
+This charter adheres to the conventions described in the [Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+## Scope
+
+The goal of this SIG is to reach and maintain a high quality of our services.
+Quality is so far defined as in the easy sentence "what helps the community and the user to solve a problem".
+Services needs to be understandable, easy to use, recreatable and doing a good job.
+High quality therefore can be defined by (not necessarly a complete or static list):
+easy to understand (know what you get before trying)
+easy to use (worth a try)
+easy to repeat (lowering the hurdle for using it again)
+solving a problem (knowing the problem)
+solving it good (knowing the community/customer)
+solving it fast (optimized, automated, workflow)
+...
+
+Following Open Source and believing in the Upstream, community is the key; as user and as contributor.
+
+Building Community and understanding problems
+Communication (with the Community) helps to find the needs or solve the problems.
+Marketing helps to understand the service and create a bigger Community.
+Metrics and Measurements (SLO and SLI) helps to do the right things and doing things right, but also fast(er).
+Governance & OrgOps helps to streamline, workflow, and bring the right people and skills to the right topics.
+
+To Tell, Measure, and Be of Service
+
+Tell: Community & Communication & Marketing
+Serve: Governance & OrgOps
+Measure: Metrics & Measurements
+
+### In scope
+Community, Communication, Marketing
+Metrics, Measurements
+Governance, OrgOps
```

Comment:
What is _OrgOps_? I think I know what you mean, but it probably deserves expanding just in case

Reply:
Needs explanation, I just wanted to use the words that people mentioned that this SIG should take care of. Let's define it.

Reply:
I still think we need to expand this a bit and explain each term here better before merging. @schwesig can you please expand on what community, communication and governance etc mean in this context? 

Also, can you please mark the pr as ready for review (remove WIP from the title)?

Reply:
tried my best, i consider it done.